src/app/course-list/models/course-item.model.ts

```diff
@@ -3,8 +3,8 @@ export interface CourseItemInterface {
     id: number;
     title: string;
     author: string;
-    creationDate: string;
-    duration: Number;
+    creationDate: Date;
```

Comment:
There should be a `string` type since it is the actual data type you get from responses.